runtimes/peregrine/src/xcm_config.rs

```diff
@@ -163,17 +181,22 @@ parameter_types! {
 }
 
 impl pallet_xcm::Config for Runtime {
-	type MaxRemoteLockConsumers = ConstU32<10>;
-	type RemoteLockConsumerIdentifier = [u8; 8];
```

Comment:
Is there a reason for this change? The spiritnet config has to be updated as well.

Reply:
Yeah I did this when I was looking at how other parachains were doing it. So it should be changed with Spiritnet as well.

Reply:
Alright will do. 

I was thinking of moving the xcm_config.rs into the runtime-common part. Spiritnet and Peregrine should be as close as possible together. I guess we want to have the same xcm config for both runtimes.

Reply:
Yes, please make sure everything else is left unchanged.